src/datasets/types.ts

```diff
@@ -28,7 +28,8 @@ export type Technology =
   | 'React'
   | 'Angular'
   | 'Rust'
-  | 'NestJS';
+  | 'NestJS'
+  | 'Next.js Flags SDK';
```

Comment:
Thinking about the sidebar a bit more, it currently mixes frameworks and programming languages.

<img width="231" alt="image" src="https://github.com/user-attachments/assets/070f958f-8d90-4756-8f87-0f65d244dc6a" />

Given this, it would be great to add Next.js and SvelteKit to the sidebar, and list our Flags SDK either simply as "Flags SDK" or "Vercel Flags SDK" for both of those categories.

We open have requests to extend the Flags SDK to more frameworks, but since we integrate deeply at the framework level it will be a while before we add another framework realistically.

Reply:
Yea ,the `Technology` section is currently overloaded by Languages + Frameworks (for example: `NestJS` / `React` / `Angular` also under `JavaScript`). Looking at this again, I don't think it makes sense for `Next.js Flags SDK` to be the Technology name, it should just be `Next.js` then if we have an official OpenFeature Next.js SDK it would also be under that Technology.

Interesting that this also works for `SvelteKit`... I'll see what I can do there.

Reply:
Updated take a look.

Reply:
This looks great now!